diff --git a/Package.swift b/Package.swift
index cedf0054079246cbe85c634aa54eec3722d6d2f3..f1bd3dfc5210d8257f27f26dd0eec74f9d0e5f11 100644
--- a/Package.swift
+++ b/Package.swift
@@ -19,10 +19,10 @@ let package = Package(
                 .product(name: "Capacitor", package: "capacitor-swift-pm"),
                 .product(name: "Cordova", package: "capacitor-swift-pm")
             ],
-            path: "ios/Sources/NostrSignerPlugin"),
+            path: "ios/Sources/NostrSignerPluginPlugin"),
         .testTarget(
             name: "NostrSignerPluginPluginTests",
             dependencies: ["NostrSignerPlugin"],
-            path: "ios/Tests/NostrSignerPluginTests")
+            path: "ios/Tests/NostrSignerPluginPluginTests")
     ]
 )
